src/components/Footer/Footer.module.css

```diff
@@ -9,3 +9,12 @@
 .tabs {
   padding-top: 20px;
 }
+
+@media (max-width: 780px) {
+  .tabs {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+} 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)